minigui/serve.py

```diff
@@ -80,21 +92,23 @@
                    "gtp.py",
                    "--load_file", args.model,
                    "--minigui_mode=true",
-                   "--num_readouts", "1000",
+                   "--num_readouts", "400",
                    "--conv_width", "128",
```

Comment:
If we're targeting something that's going to use our current models, the default has been 256 at least since v7.  If the default case is gonna be the edge TPU stuff, we can leave it at 128.